### PR TITLE
Ignore the `:label_text` when rendering `label` tags.

### LINKED ISSUE #1314: SimpleForm leaks the formatting function into HTML
When overriding the `label_text` per input like so:

```
form.label :name, label_text: proc { |label_text, required, _| "#{label_text} #{required}" }
```

The generated HTML has:

```
<label label_text="#<Proc:0x007fe268b21ce8@/Users/foo/boo/_new_form.html.slim:4>" class="string required" for="user_attributes_name">Name <abbr title="required">*</abbr></label>
```

As you can see it includes the full path to the custom proc function location which is a security issue.

This happens with versions `3.1.0` and `3.1.1.`

\cc @dnagir

### PR DESCRIPTION
**Pull request by lucasmazza**
Closes #1314.

### CODE REVIEW

File: lib/simple_form/components/labels.rb [hunk 1]
```diff
@@ -50,7 +53,7 @@ def label_html_options
           label_options[:for] = options[:input_html][:id]
         end
 
-        label_options
+        label_options.except(*VIRTUAL_OPTIONS)
```
Comment (rafaelfranca) [inline lib/simple_form/components/labels.rb]: Did you see why it is here in the first place? `html_options_for(:label)`  should only return things that are in the `label_html` key.

Comment (rafaelfranca) [inline lib/simple_form/components/labels.rb]: Oh, we should be filtering it here https://github.com/plataformatec/simple_form/blob/81b9d7d3439225c7613e570afb44596acfa97a76/lib/simple_form/form_builder.rb#L302

Comment (lucasmazza) [inline lib/simple_form/components/labels.rb]: :+1:

File: test/form_builder/label_test.rb [hunk 2]
```diff
@@ -120,4 +120,12 @@ def with_label_for(object, *args, &block)
       assert_select 'label[for=user_time_zone]', 'Time Zone:'
     end
   end
+
+
```
Comment (rafaelfranca) [inline test/form_builder/label_test.rb]: ✂️

**Comment by rafaelfranca** [comment]
:shipit:

**Pull request merged**